### PR TITLE
Update DeepProfiler Processing

### PR DESCRIPTION
**Pull request by michaelbornholdt**
Addresses some of the problems from issue #144
 
# Description

Thank you for your contribution to pycytominer!
Please _succinctly_ summarize your proposed change.
What motivated you to make this change?

Please also link to any relevant issues that your code is associated with.

## What is the nature of your change?

- [ ] Bug fix (fixes an issue).
- [x] Enhancement (adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

# Checklist

Please ensure that all boxes are checked before indicating that a pull request is ready for review.

- [x] I have read the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have deleted all non-relevant text in this pull request template.

### CODE REVIEW

**Comment by michaelbornholdt** [comment]
@gwaygenomics 
@niranjchandrasekaran 
This is a quick update on the DeepProcessing. 

Would be great if Niranj can have a look. The whole function is a lot nicer now.

File: pycytominer/cyto_utils/DeepProfiler_processing.py [hunk 1]
```diff
@@ -190,27 +200,19 @@ def aggregate_deep(self):
...
-            meta_df = pd.DataFrame(
-                self.file_aggregate[metadata_level]["metadata"], index=[0]
-            ).reset_index(drop=True)
-            meta_df.columns = [
-                f"Metadata_{x.capitalize()}" if not x.startswith("Metadata_") else x
-                for x in meta_df.columns
-            ]
+            # extract metadata prior to aggregation
+            meta_df = pd.DataFrame()
+            metadata_cols = [x for x in df.columns if x.startswith("Metadata_")]
```
Comment (niranjchandrasekaran) [inline pycytominer/cyto_utils/DeepProfiler_processing.py:207]: You could call `infer_cp_features(metadata=True)` from `pycytominer.cyto_utils()`

**Review by niranjchandrasekaran — COMMENTED** [review]
Looks good @michaelbornholdt. But we should wait for #150 to be fixed before merging this.

**Comment by niranjchandrasekaran** [comment]
Now that #150 has been merged, @michaelbornholdt after your push your next commit (or push a dummy commit), it will trigger GitHub actions and run pytest. Once it passes all the tests, I will approve the PR.

**Comment by michaelbornholdt** [comment]
@niranjchandrasekaran Looking good :)

**Review by niranjchandrasekaran — APPROVED** [review]
👍 LGTM

**Comment by michaelbornholdt** [comment]
Should I be merging then? @niranjchandrasekaran

**Comment by shntnu** [comment]
Can we delete this branch? @niranjchandrasekaran

**Comment by niranjchandrasekaran** [comment]
Yes!

**Pull request merged**